File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/non-plagiarized/06/T04.ts
import { Scanner } from 'util';

/**
 *
 * @author 92E0988C1682C76D4D307AA15EC8346E
 */
const milToKilo: number = 1.609;

console.log('Miles\tKilometers');
for (let i: number = 1; i < 11; i++) {
  const kilo: number = i * milToKilo;
  console.log(`${i}\t${kilo}`);
}

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/non-plagiarized/06/T04.kt
package IR

import java.util.Scanner

/**
 *
 * @author 92E0988C1682C76D4D307AA15EC8346E
 */
fun main() {
    val milToKilo = 1.609
    println("Miles\tKilometers")
    for (i in 1..10) {
        val kilo = i * milToKilo
        println("$i\t$kilo")
    }
}

